roles/openshift_logging/templates/fluentd.j2

```diff
@@ -59,6 +59,11 @@ spec:
         - name: dockercfg
           mountPath: /etc/sysconfig/docker
           readOnly: true
+{% if openshift_logging_use_mux_client %}
```

Comment:
fluentd collector needs the mux ca cert and mux shared_key for the client side of secure_forward - see https://github.com/openshift/origin-aggregated-logging/pull/378/files#diff-5c5eff90645c105043514f4377984270